todo_app/templates/index.html

```diff
@@ -7,17 +7,23 @@ <h1 class="display-4">To-Do App</h1>
     <p class="lead">Just another to-do app.</p>
   </div>
 
+  <form action="" method="get" class="form-example">
+    <div class="form-title">
+      <label for="title">Enter your title: </label>
+      <input type="text" name="name" id="name" required>
+      <button class="favorite styled"
+        type="button">
+          Submit
+      </button>
+    </div>
+  </form>
+ 
+
   <div class="row">
     <div class="col-md-12">
       <h2>Items</h2>
       <ul class="list-group mb-4">
-        <li>SprintPlanning</li>
-        <li>Refactoring</li>
-        <li>CodeReviewing</li>
         <table>
-          <tr>
-            <th>Books List</th>
-          </tr>
           {% for item in list %}
           <tr>
             <td> {{item}} </td> 
```

Comment:
You should try controlling what you display, by accessing e.g. `item.title`. You are currently displaying this on the page, which isn't ideal:

```
{'id': 1, 'status': 'Not Started', 'title': 'List saved todo items'}
{'id': 2, 'status': 'Not Started', 'title': 'Allow new items to be added'}
```